free-3.md

```diff
@@ -57,12 +116,40 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Implementing the visualization aspect might be difficult. I'm not familiar with
+what solutions are already out there for generating images, so I until I am
+able to look into it further it will be difficult to say exactly how much of a
+challenge this will be.
+
+The bigger difficulty would be implementing the interactive features a la
+Victor's system. However, those are features that could be added after
+achieving a working prototype.
+
+Much of the rest of the time spent designing the language would be in designing
+the conversion from formulas to images. Getting this right will be both
+difficult and critical, because I suspect that not any implementation will
+yield interesting visual results. However, I would argue that making these
+decisions is equivalent to designing the language itself, and not the
+implementation, because the decisions will be heavily driven by the need to
+provide an intuitive and rewarding interface to the user.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think it's borderline. It seems like a difficult project, but as I described
+above, it could be implement in stages. A minimum viable product might not
+include the interactive parameter adjustment I talked about; it might function
+just as a way to visualize mathematical expressions, with the more interactive
+features being added later as time permits.
```

Comment:
I agree. Interactiveness is a fairly difficult thing to have implemented within the time limit we have and all the awesome things we want in our DSLs.